legacy_promotions/db/migrate/20160101010001_solidus_one_four_promotions.rb

```diff
@@ -1,8 +1,8 @@
 # frozen_string_literal: true
 
-class SolidusOneFourPromotions < ActiveRecord::Migration[5.0]
+class SolidusOneFourPromotions < ActiveRecord::Migration[6.1]
```

Comment:
The reason I changed to `ActiveRecord::Migration[6.1]` is because Rails 6.1 introduces the `if_exists:` and `if_not_exists` flags. These allow us to run the migration and get fine-grained control over what they do even if older migrations have done parts of what these migrations do. 

All of these migrations will be *new* migrations for all Solidus users, and they really should be no-ops if those tables already exist. Given this will only be released with Solidus 4.4, we can reasonable assume that people are already on Rails 7, so a migration version of 6.1 should not be a problem.

Reply:
I saw why you introduce that new version, and it's clear that those will be executed only for new stores. 

My point is that if we have a new migration in the future that interacts with what's generated with those migrations, we have no certainty about which Rails version has been used to create those tables. Hope that's more clear now, but let me know if still doesn't make sense.

Reply:
I can change this commit to using explicit `if` statements instead of the `if_not_exists` and keep the migration version.

Reply:
Done!